modules/upload-definitions-to-central-admin-pass.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * configuration/enable-offline-mode.adoc
+:_module-type: PROCEDURE
+[id="upload-definitions-to-central-admin-pass_{context}"]
+= Uploading definitions to Central by using the administrator password
+
+You can upload the vulnerability definitions database (that Scanner uses) to Central by using your {product-title} administrator password.
+
+.Prerequisites
+
+* You must have the administrator password.
+* You must have installed the `roxctl` command-line interface (CLI).
+
+.Procedure
+
+. Set the `CENTRAL_ADDRESS` token:
```

Comment:
Just a question, is this really a token? Or could it be called just an environment variable?

(Same for below module too)